core/data/src/main/java/com/google/samples/apps/nowinandroid/core/data/repository/OfflineFirstNewsRepository.kt

```diff
@@ -72,15 +74,25 @@ class OfflineFirstNewsRepository @Inject constructor(
             },
             modelDeleter = newsResourceDao::deleteNewsResources,
             modelUpdater = { changedIds ->
+                val userData = niaPreferencesDataSource.userData.first()
```

Comment:
Should we handle `NoSuchElementException` here?

Reply:
We shouldn't need to, the proto3 definition implicitly has default values so datastore should always return a value.